### PR TITLE
[OMI-597] Wrap AMCustomEventBanner

### PR DESCRIPTION
**Pull request by can-soykarafakili**
I don't think that this ticket requires any more work since the necessary codes will be added AMCustomEventBanner classes for each integration type

### CODE REVIEW

**Comment by eros902002** [comment]
You're trying to merge to development : )

**Comment by eros902002** [comment]
The flower name for the MoPub adapters also has a small typo.

**Comment by can-soykarafakili** [comment]
Thanks for the catches @eros902002! Updated the folder name & destination brach

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/AdMob Mediation Adapter/AMCustomEventBanner.m [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AMCustomEventBanner.h"
+
+@implementation AMCustomEventBanner
```
Comment (eros902002) [inline PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/AdMob Mediation Adapter/AMCustomEventBanner.m:25]: Don't we need to map the sizes here from theirs to ours?

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/MoPub Mediation Adatper/AMCustomEventBanner.m [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AMCustomEventBanner.h"
```
Comment (eros902002) [inline PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/MoPub Mediation Adatper/AMCustomEventBanner.m:23]: Still a small typo in the folder of the file : )

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/AdMob Mediation Adapter/AMCustomEventBanner.m (hunk 1 above)
Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/AdMob Mediation Adapter/AMCustomEventBanner.m:25]: I'll do that when I work on the AdMob Mediation. That size mapping is also required for MoPub Mediation as well i think 👍

File: PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/MoPub Mediation Adatper/AMCustomEventBanner.m (hunk 2 above)
Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/MoPub Adapters/MoPub Mediation Adatper/AMCustomEventBanner.m:23]: Apparently I can not type "adatper" today. 🤦  Oh wait... 😂

**Review by eros902002 — APPROVED** [review]
: )

**Pull request merged**